refactor/urls.js

```diff
@@ -1,16 +1,17 @@
-// Refactor this code
+const TASKS_BASE = "/tasks";
```

Comment:
Why is this a constant which is not being used again ? Can we not declare url as /tasks